bower.json

```diff
@@ -2,17 +2,23 @@
   "name": "fxa-content-server",
   "version": "0.0.0",
   "dependencies": {
+    "JavaScript-MD5": "1.1.0",
```

Comment:
Capitals before Small? Fixpack seems to do it that way, you want me to change it?